CleanTalk/XF/Template/Templater.php

```diff
@@ -121,12 +121,28 @@ public function renderTemplate($template, array $params = [], $addDefaultParams
 			{
 				CleantalkFuncs::sfwCheck();  				
 			}
+
+            if ( $_SERVER["REQUEST_METHOD"] == 'GET' && $_SERVER['SCRIPT_NAME'] === '/admin.php' ) {
+                //key error render
+                $key_error = $this->app->options()['ct_apikey_error'];
```

Comment:
does the object exactly have such a key in the array? Maybe check it out: isset($this->app->options()['ct_apikey_error'])

Reply:
Done. https://github.com/CleanTalk/xenforo2-antispam/commit/66cf557ddb46cc6a37349d027633fea783c47f29